### ISSUE TITLE
Don't work on anki 2.1.20

### ISSUE THREAD

**Issue by ilovlowphy** [tsudoko/anki-sync-server#56]
Don't work on anki 2.1.20. When I go back 2.1.15, it gets back normal.

**Comment by bosconb** [comment]
2.1.20 in mac, everything is ok for me so far

**Comment by Junyu25** [comment]
Yeah, I got error like this when I update to Anki 2.1.20
@tsudoko I don't know whether if it's because I have a large deck, or it's incompatible with Anki2.1.20.
![Snipaste_2020-03-05_09-07-18](https://user-images.githubusercontent.com/22055160/75937997-7d2cab80-5ec1-11ea-9fc3-7ef1f108702a.png)

**Comment by mrozigor** [comment]
I was able to run Anki 2.1.20 and sync it with anki-sync-server, but probably only data (no media). It looks like it tries to sync media with ankiweb, regardless of what is set in SYNC_BASE variable. When I put my server IP in method that creates media sync URL (in Anki), then anki-sync-server rejects request (403; it looks like the session isn't initialized earlier).

**Comment by Junyu25** [comment]
@mrozigor You are right, I can sync the card data, but when it comes to media, it failed.
So I `Check Media` file and find out there are some media use it card but not in my media folder.
After I delete the card field that contain the missing media, it sync successfully.
Note that I reinstall the anki-sync-server and also install the pyaudio to check if is required for anki 2.1.20.

**Comment by mrozigor** [comment]
I hope that my [PR](https://github.com/tsudoko/anki-sync-server/pull/60) will help with some of the issues.
